src/languages/index.js

```diff
@@ -70,4 +70,24 @@ function translateLanguage(key, params = {}) {
   return interpolateText(translation, params);
 }
 
-module.exports = { translateLanguage };
+const translateCommand = (command) => {
+  const languages = {
+    'es-ES': 'es',
+    'en-US': 'en',
+  };
+
+  const result = {};
+
+  for (const [locale, lang] of Object.entries(languages)) {
+    const translations = loadTranslations(lang);
+    if (translations) {
+      result[locale] = getTranslationByKey(translations, command, lang);
+    } else {
+      result[locale] = `Missing translation for ${lang}`;
+    }
```

Comment:
This logic is the same that have translateLanguage function, but also we are missing the interpolateText feature here. In this case would be better to use translateLanguage instead of repeating logic, since if translateLanguage change we would need to made same changes here, repeat the same changes in two different places is something we should always avoid

Reply:
You can refactor translateLanguage to accept the language by parameters, for example:

```js
function translateLanguage(key, params = {}, language = botLanguage) {
  const translations = loadTranslations(language);
  if (!translations || !Object.values(translations).length) {
    return `Translations not available for language '${botLanguage}'.`;
  }
  const translation = getTranslationByKey(translations, key, botLanguage);
  return interpolateText(translation, params);
}
```

then use it here